### PR TITLE
[Trilinos][FluidDynamics][FluidDynamicsBiomedical] Migrating Tests

### PR DESCRIPTION
**Pull request by roigcarlo**
**📝 Description**
This is the First PR in a series of PR which will remove the tests suits from the core to the applications themselves.
I will try to split them down to 2-3 applications per batch that fall on the same field.

In this case
- FluidDynamics
- FluidDynamicsBiomedical
- TrilinosApplication (needed because of the mpi extension)

### CODE REVIEW

File: applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+//  Main authors:    Richard Faasse
+//                   Carlos A. Roig
+//
+
+#pragma once
+
+#include "testing/testing.h"
+#include "fluid_dynamics_application.h"
+
+namespace Kratos::Testing {
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.h]:
```suggestion
namespace Kratos::Testing
{
```

File: applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.h [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+class FluidDynamicsApplicationFastSuite : public KratosCoreFastSuite {
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.h]:
```suggestion
class FluidDynamicsApplicationFastSuite : public KratosCoreFastSuite
{
```

File: applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.h [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+public:
+  FluidDynamicsApplicationFastSuite();
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.h]:
```suggestion
    FluidDynamicsApplicationFastSuite();
```

File: applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.h [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+private:
+  KratosFluidDynamicsApplication::Pointer mpFluidDynamicsApp;
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.h]:
```suggestion
    KratosFluidDynamicsApplication::Pointer mpFluidDynamicsApp;
```

File: applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.cpp [hunk 5]
```diff
@@ -11,25 +11,15 @@
 //                   Carlos A. Roig
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.cpp:11]: Do we strictly require the source file?

Comment (roigcarlo) [reply applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.cpp:11]: Not strictly, but its better to keep it consistent with other apps that may need to do extra steps to initialize and also prevent possible empty suits from failing to compile.

Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.cpp:11]: Fair enough. Fine for me then.

File: applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.cpp [hunk 6]
```diff
@@ -11,25 +11,15 @@
...
+#include "fluid_dynamics_fast_suite.h"
 
-    Kratos::Testing::mApplicationInitializerList.push_back([](std::vector<Kratos::KratosApplication::Pointer> & rRegisteredApplications, Kratos::Kernel & rKernel) {
-      if (!rKernel.IsImported("FluidDynamicsApplication")) {
-        auto pApplication = std::make_shared<Kratos::KratosFluidDynamicsApplication>();
-        rKernel.ImportApplication(pApplication);
-        rRegisteredApplications.push_back(std::move(pApplication));
-      }
-    });
+namespace Kratos::Testing {
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.cpp]:
```suggestion
namespace Kratos::Testing
{
```

File: applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.cpp [hunk 7]
```diff
@@ -11,25 +11,15 @@
...
 
-    return RUN_ALL_TESTS();
+FluidDynamicsApplicationFastSuite::FluidDynamicsApplicationFastSuite()
+    : KratosCoreFastSuite() {
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.cpp]:
```suggestion
    : KratosCoreFastSuite()
{
```

File: applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.cpp [hunk 8]
```diff
@@ -11,25 +11,15 @@
...
+  mpFluidDynamicsApp = std::make_shared<KratosFluidDynamicsApplication>();
+  this->ImportApplicationIntoKernel(mpFluidDynamicsApp);
```
Comment (rubenzorrilla) [inline applications/FluidDynamicsApplication/tests/cpp_tests/fluid_dynamics_fast_suite.cpp]:
```suggestion
    mpFluidDynamicsApp = std::make_shared<KratosFluidDynamicsApplication>();
    this->ImportApplicationIntoKernel(mpFluidDynamicsApp);
```

**Review by rubenzorrilla — APPROVED** [review]

**Pull request merged**